impexp-client-gui/src/main/java/org/citydb/gui/operation/common/filter/XMLQueryView.java

```diff
@@ -414,6 +431,32 @@ private String wrapQuery(String query) {
         return wrapper.toString();
     }
 
+    private void createSQLQuery() {
+        if (!ObjectRegistry.getInstance().getDatabaseController().connect()) {
+            return;
```

Comment:
Shouldn't we add a confirmation dialog here that a connection to the database must be established? There should also be a corresponding log entry.

Reply:
see db2b5ba